File: w09/d03/katherine_santiago/mail_objects_app/spec/javascripts/mailitem_spec.js
describe("a mailitem", function() {

  beforeEach(function() {
    mailitem = new mailItem("New York", .50);
  });

  it("is created with a shipToAddress", function() {
    expect(mailitem.shipToAddress).toBe("New York");
  });

  it("has a stampValue when created", function() {
    expect(mailitem.stampValue).toBe(.50);
  });
});

File: w09/d01/jimmy_davis/mail_items/spec/mailitem_spec.rb
require_relative "spec_helper"
require_relative "../lib/mailitem"

describe MailItem do

  it "should have a ship to address" do
    mailitem = MailItem.new
    expect(mailitem.ship_to_address).to be_true
  end

  it "should have a stamp value" do
    mailitem = MailItem.new
    expect(mailitem.stamp_value).to be_true
  end

end
